### PR TITLE
Jhinchley/add order

### PR DESCRIPTION
**Pull request by hinchley2018**
#5 Add Order functionality

### CODE REVIEW

**Comment by CodeSTACKE** [comment]
added the functionality  of the add button, when clicked.

File: src/user-utilities.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
     //adds order to user's orders {id: Number, productDescription: string, quantity: Number}
     function addOrder(productDescription, quantity) {
-        
+        let myOrder = {id: user.orders.length, productDescription, quantity}
+        user.orders.push(myrder)
```
Comment (omarmoreta) [inline src/user-utilities.js]: myOrder in the ,push(  ) is not correctly spelled.

**Review by omarmoreta — COMMENTED** [review]
Look into our review and change as needed, let us know if you need any help or have any questions.
Sriya
Derek
Carlos
Jyoti
Omar

File: src/user-utilities.js (hunk 1 above)
Comment (hinchley2018) [reply src/user-utilities.js]: Done

**Comment by hinchley2018** [comment]
closes #5

**Pull request merged**